meetings/2023-11-08.md

```diff
@@ -0,0 +1,183 @@
+* SC: Regarding the PR, I want to acknowledge an honor Henry. I want to know if people are ok with what I proposed.
+* AC: I am +1 on SC's change
+* AC: It is important to recognize Henry's contributions. At the same time it is important to have a clear person who is responsible for moving the work item forward.
+* SC: That was the idea to signal who is the contact person now.
+
+
+### Remove contributors, add authors
+URL: https://github.com/solid/web-access-control-spec/pull/126
+
+* SC: Any objections or suggestions?
+* SC: There are only few specs which mention contributors, they usually have editors and authors.
+* SC: I asked Henry long time ago can I add him as author. He didn't give me a clear answer.
+* eP: IIRC, Henry was talking about WAC+ and had various functionality. Maybe he didn't want to be listed as author there was no consensus to include his changes so perhaps speculative but maybe he was talking about it. How ACP's stuff could also work in WAC re same use cases, evaluations .. https://github.com/solid/authorization-panel/tree/main/proposals/evaluation
+* SC: I talked with Matthieu and actually some things in ACP were improved based on Henry's feedback.
+
+
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
+
+* SC: Notes: https://hackmd.io/a6jv3bfnQBGace7Ajd_ePw
+* SC: I was talking with W3C ... We don't have complete support as it would be for TAG or AC elections.
+* SC: We have some code that gets data from W3C API
+* SC: https://github.com/VirginiaBalseiro/solid-ecosystem-monitor/pull/2
+* SC: The idea is that by default we take participant with lowest w3c id (which is incremental number)
+* SC: We will offer all affiliations to change a name.
+* SC: In the end we will give a list of eligible voters to W3C staff
+* AC: That sounds fine
+* SC: While you here maybe you can give heads up to Inrupt since it has 14 participants.
+* AC: I would probably be the inrupt name, you could use my name as default/designated Inrupt representative.
+* eP: I will add you as designated and link to the minutes.
+* AC: Whatever is easiest for your tooling is fine.
+
+### Align shared core terminology
+URL: https://github.com/solid/specification/discussions/557
+
+* SC: The discussion served its purpose (but we can improve as suggested by Ted). We should considering convert this to issues and/or PRs. And try to cover them for the next releases of the specs.
+* SC: We need to be careful with what terminology we are changing. We could move that into small PRs and issues.
+* eP: We may also want to draw a balance and provide a documentation where they can be input for the WG. Not sure if PRs would be best way because we hope it'll be transitioned to WG.
+* SC: I think PR or any input we make matters. I hope that WG will need to review all the material. If there is a text out there I try to process it. PR is more about let's make a decision. The WG can always take a different direction.
```

Comment:
```suggestion
* SC: The discussion served its purpose (but we can improve as suggested by Ted). We should consider converting this to issues and/or PRs. And try to cover them for the next releases of the specs.
* SC: We need to be careful with the terminology we are changing. We could move that into small PRs and issues.
* eP: We may also want to draw a balance, and provide documentation where they can be input for the WG. Not sure if PRs would be best way, because we hope it'll be transitioned to WG.
* SC: I think PR or any input we make matters. I hope that WG will review all the material. If there is a text out there, I try to process it. PR is more about "let's make a decision." The WG can always take a different direction.
```